Kernel/Devices/Serial/NS16550/PCINS16550.h

```diff
@@ -9,14 +9,14 @@
 #include <Kernel/Bus/PCI/Device.h>
 #include <Kernel/Bus/PCI/IDs.h>
 #include <Kernel/Devices/CharacterDevice.h>
-#include <Kernel/Devices/SerialDevice.h>
+#include <Kernel/Devices/Serial/NS16550/NS16550.h>
 
 namespace Kernel {
 
-class PCISerialDevice {
+class PCINS16550 {
```

Comment:
I'm not really happy with the name "PCINS16550", it would be nice if the word gap between "PCI" and "NS16550" would be more visible. "PCIAttachedNS16550" might be an option, but still looks a bit silly.

The "NS" prefix of "NS16550" is necessary, since identifiers can't start with numbers. (NS because National Semiconductor, similar to the [devicetree compatible string name](https://github.com/devicetree-org/devicetree-specification/blob/ba20321b75918289554653a41c178104fe29c182/source/chapter4-device-bindings.rst?plain=1#L182))

Reply:
Another option might be calling these classes `Serial16550` and `PCISerial16550` or `Serial16550Device` and `PCISerial16550Device`.

Reply:
If you're looking for bikeshed input, I like `PCISerial16550` :)